app/models/measurement.rb

```diff
@@ -17,4 +17,28 @@ class Measurement < ActiveRecord::Base
   def prefix_and_name
     "#{prefix} -> #{data_portal_name}"
   end
+
+  def Measurement.create_new(properties)
+    properties = properties.map {|k,v| [k.to_sym, v.blank? ? nil : v] }.to_h  ## don't put empty strings in the db
+    raise 'No prefix specified' unless properties[:prefix]
+    Measurement.create(properties)
+  end
+
+  def add_series(series)
+    self.transaction do
+      series.update_attributes(universe: 'DBEDTCOH') if universe == 'DBEDTCOH'
```

Comment:
This type of manipulation makes a series_universe join table seem more attractive